processing/database/__init__.py

```diff
@@ -1 +1,5 @@
-MUNICIPALITIES = {'4314902': 'RsPortoAlegre', '5208707': 'GoGoiania'}
+MUNICIPALITIES = {
+    '4314902': 'RsPortoAlegre',
+    '5208707': 'GoGoiania',
+    '2910800': 'BaFeiraDeSantana',
```

Comment:
Can you remove this line? It will make the parser try to search for a `BaFeiraDeSantana`. Won't raise any exception (anymore) but it's not necessary.